client/src/components/Article/Article.tsx

```diff
@@ -1,7 +1,6 @@
 import React from 'react';
 import styled from 'styled-components';
-import { Link } from 'react-router-dom';
-import { Icon, themeColor, themeSpacing } from '@amsterdam/asc-ui';
+import { Icon, themeColor, themeSpacing, breakpoint, Link } from '@amsterdam/asc-ui';
```

Comment:
Hier ruil je Link uit react-router-dom in voor een gewone li toch? Weet niet of dat de bedoeling is.

Reply:
Ja, dat komt omdat de ASC links niet underlined zijn  by default. Maar denk inderdaad mooier om react-router-dom te gebruiken dus pas het ff aan.